### PR TITLE
updated header text to template

### PR DESCRIPTION
**Pull request by jessicabartley**
Changed some of the language in the intro template upon request by @angielaird

### CODE REVIEW

File: _layouts/post.html [hunk 1]
```diff
@@ -17,14 +17,14 @@ <h1 class="page-title">{{page.title}}</h1>
...
-                This week's Instructor Q&A session has been completed.
+                This week's Instructor Q&A session has ended.
                 Please see the bottom of this page for the recording of the session.
                 </div>
 
                 <div class="bigspacer"></div>
 
                 <div class="smallhead">
-                <b><a class="off" href="{{ page.qa_doc }}">{{page.title}} Q&A Doc</a></b>: This document is frozen, but contains answers to the presubmitted Q&A.
+                <b><a class="off" href="{{ page.qa_doc }}">{{page.title}} Q&A Doc</a></b>: Click here for answers to the Q&A Doc.
```
Comment (tsalo) [inline _layouts/post.html:27]: Doesn't "Click here" imply that the hyperlink will be on the word "here"?

Comment (jessicabartley) [reply _layouts/post.html:27]: Sorry about that. I suppose so, yes. I'll update to include a link for "here"

File: _layouts/post.html [hunk 2]
```diff
@@ -17,14 +17,14 @@ <h1 class="page-title">{{page.title}}</h1>
...
-                This week's Instructor Q&A session has been completed.
+                This week's Instructor Q&A session has ended.
                 Please see the bottom of this page for the recording of the session.
                 </div>
 
                 <div class="bigspacer"></div>
 
                 <div class="smallhead">
-                <b><a class="off" href="{{ page.qa_doc }}">{{page.title}} Q&A Doc</a></b>: This document is frozen, but contains answers to the presubmitted Q&A.
+                <b><a class="off" href="{{ page.qa_doc }}">{{page.title}} Q&A Doc</a></b>: Click <a class="off" href="{{ page.qa_doc }}">here/a> for answers to the Q&A Doc.
```
Comment (tsalo) [inline _layouts/post.html:27]:
```suggestion
                <b>{{page.title}} Q&A Doc</b>: Click <a class="off" href="{{ page.qa_doc }}">here</a> for answers to the Q&A Doc.
```

**Review by tsalo — CHANGES_REQUESTED** [review]
I'd like to remove the duplicate hyperlink and there was a small issue with the one you added (no opening `<` in the closing `</a>`).

File: _layouts/post.html (hunk 2 above)
Comment (jessicabartley) [reply _layouts/post.html:27]: On it!

**Review by tsalo — APPROVED** [review]
LGTM.

**Pull request merged**